<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_deviceLog</class>
 <widget class="QDialog" name="Dialog_deviceLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>21</y>
     <width>311</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>事件</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
